spec/stored_queries/generic/fulltext_search.yaml

```diff
@@ -1,94 +0,0 @@
-# Should be REVISED or DEPRECATED.
```

Comment:
I wrote this and I know it's not being used because of performance issues so axed the cruft